### ISSUE TITLE
Fix fractional energy loss

### ISSUE THREAD

**Issue by evann-lbl** [LBNL-ETA/ROAMS#6]
The numerator fractional energy loss is currently only defined as the energy content of CH4 emitted, while the denominator is the energy content of both oil and gas produced considering the gas composition of produced NG.

The emitted CH4 should be backwards-engineered into a quantity of NG using a provided gas composition, so that the total energy content of the NG can be estimated.

**Comment by evann-lbl** [comment]
The numerator of fractional energy loss is currently just CH4. The computation appears in several parts of the codebase (you can look for `ENERGY_DENSITY_MJKG["c1"]`), and always goes something like this:

1. Energy of CH4 = mass of ch4 emitted * energy density of CH4 = numerator
2. Denominator = energy content of produced NG (using gas composition) and oil
3. Fractional energy loss = (1)/(2)

This has to be converted in all these places to something like this:

1. Volume of emitted CH4 = mass of emitted CH4 / density of CH4
2. Volume of emitted NG = volume of emitted CH4 / fraction of NG that is CH4
3. Energy of emitted NG = energy content of (3) using gas composition
4. Denominator = energy content of produced NG (use of gas composition) and oil
5. Fractional energy loss = (3)/(4)
